.github/workflows/lint-js-and-ruby.yml

```
@@ -45,12 +45,29 @@ jobs:
+      run: cd spec/dummy && yarn install --no-progress --no-emoji
     - name: Install Ruby Gems for package
       run: bundle check --path=vendor/bundle || bundle _2.5.9_ install --path=vendor/bundle --jobs=4 --retry=3
     - name: Lint Ruby
       run: bundle exec rubocop
     - name: Install Node modules with Yarn for dummy app
       run: cd spec/dummy && yarn install --ignore-scripts --no-progress --no-emoji
+    - name: Save dummy app ruby gems to cache
+      uses: actions/cache@v4
+      with:
+        path: spec/dummy/vendor/bundle
+        key: dummy-app-gem-cache-${{ hashFiles('react_on_rails.gemspec') }}-${{ hashFiles('Gemfile.development_dependencies') }}-${{ matrix.versions }}
```

actionlint (1.7.4)

property "versions" is not defined in object type {}

(expression)